nexus/db-queries/src/db/datastore/bgp.rs

```diff
@@ -606,6 +606,7 @@ impl DataStore {
             .transaction(&conn, |conn| async move {
                 let active = peer_dsl::switch_port_settings_bgp_peer_config
                     .filter(db_peer::port_settings_id.eq(port_settings_id))
+                    .filter(db_peer::addr.eq(addr))
```

Comment:
Huh. We had this filter for communities above at line 583, but missed it for import/export filters.